Gemfile

```diff
@@ -84,6 +84,7 @@ group :development, :test do
   # Call 'byebug' anywhere in the code to stop execution and get a debugger console
   # or call 'binding.pry'  (you may need require 'pry-byebug' first)
   gem 'pry-byebug'
```

Comment:
Can we remove `pry-byebug` if we're using `debug` instead?

Reply:
for sure!